### DISCUSSION TITLE
Release 2.1.1

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [Whisky-App/Whisky#570] [Announcements]
**New Features:**
- New settings window to allow you to configure settings like Whisky's update behaviour
- WhiskyKit is now a Swift Package, allowing it to be integrated into other projects easily

**Quality of Life:**
- Removed some unused dependencies
- All in-app internet requests will now be made to Whisky's server, eliminating the need to use a VPN to update Whisky for users in mainland China
- Better in-app help menu with quick links for Whisky's Website, Discord, and GitHub

**Bug Fixes:**
- Significantly reduced the memory footprint of Whisky
- Fixed a crash that would occur with certain programs and bottles

**Full Changelog:** https://github.com/Whisky-App/Whisky/compare/v2.1.0...v2.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Whisky-App/Whisky/releases/tag/v2.1.1'>Release 2.1.1</a>.</em>

**Comment by OldTabdy** [comment]
D2 Resurrected still isn't working on the new release. The last version that successfully runs D2R is 2.02.
